src/backend/logger/logger.ts

```diff
@@ -507,7 +635,11 @@ class GameLogWriter extends LogWriter {
 
       // log game settings
       const gameSettings = await gameManagerMap[runner].getSettings(app_name)
-      const gameSettingsString = JSON.stringify(gameSettings, null, '\t')
+      const gameSettingsString = JSON.stringify(
+        this.filterGameSettingsForLog({ ...gameSettings }, notNative),
```

Comment:
```suggestion
        this.filterGameSettingsForLog(structuredClone(gameSettings), notNative),
```
Right now, this will only shallow-clone the settings object